cmd/release/cmd/root.go

```diff
@@ -32,26 +39,40 @@ func SetVersion(version string) {
 }
 
 func init() {
-	rootCmd.PersistentFlags().BoolP("debug", "d", false, "Debug")
-
-	configPath, err := config.DefaultConfigPath()
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
-	}
+	rootCmd.PersistentFlags().BoolVarP(&debug, "debug", "D", false, "Debug")
```

Comment:
What's the value / reason for changing the d to upper case?

Reply:
There isn't any big value on changing it, the main reason is to maintain consistency with the dry run flag. Neither are being used by any sub-cmds yet